plugins/src/webrtcsink/imp.rs

```diff
@@ -325,6 +326,16 @@ fn make_converter_for_video_caps(caps: &gst::Caps) -> Result<gst::Element, Error
                 gst::Element::link_many(&[&glupload, &glconvert, &glscale])?;
 
                 (glupload, glscale)
+            } else if feature.contains(NVMM_MEMORY_FEATURE) {
+                let queue = make_element("queue", None)?;
+                let nvconvert = make_element("nvvideoconvert", None)?;
+                nvconvert.set_property_from_str("compute-hw", "VIC");
```

Comment:
keeping this at 0 should automatically pick the correct hw no?

Reply:
It should but I have seen that sometimes it does not work in the best way...but for portability matters in webrtcsink I agree with you and that it should be 0.

Reply:
I'm afraid I can't merge this if it caters only to your use case and leaves half of the potential user base in the cold :( Please set these properties to 0 and report any issue you may have with it against the nvidia elements.

Reply:
oh my bad you actually ended up changing it, thanks :)